Automattic/jetpack
# Deprecate soft disconnect

#### Changes proposed in this Pull Request:

* Deprecates Soft Disconnect

#### Jetpack product discussion


p9dueE-4n9-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Nothing should change, because we are not really using Soft disconnection. This PR only removes and deprecates code that is not actively in use and updates the documentation.

Test connection, disconnection and reconnection flows.
